test/framework/shootmigrationtest.go

```diff
@@ -160,7 +162,7 @@ func (t *ShootMigrationTest) MigrateShoot(ctx context.Context) error {
 	}
 
 	t.MigrationTime = metav1.Now()
-
+	fmt.Printf("========= Migrating shoot ============")
```

Comment:
What is this?
Use `By` for prominently displaying test steps, otherwise never use `fmt.Print*`!?